src/App.tsx

```diff
@@ -11,7 +11,7 @@ import { Layout } from "./Layout";
 // Bring in the pages
 import { Home } from "./pages/Home";
 import { About } from "./pages/About";
-import { DataRoute } from "./pages/DataRoute";
+// import { DataRoute } from "./pages/DataRoute";
```

Comment:
I tend to just delete things since you can always go back in git history instead of leaving commented out code.